x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -172,7 +172,7 @@ class PackagePolicyService implements PackagePolicyServiceInterface {
       if (isPackageLimited(pkgInfo)) {
         if (agentPolicy && doesAgentPolicyAlreadyIncludePackage(agentPolicy, pkgInfo.name)) {
           throw new IngestManagerError(
-            `Unable to create package policy. Package '${pkgInfo.name}' already exists on this agent policy.`
+            `Unable to create agent policy. Integration '${pkgInfo.name}' already exists on this agent policy.`
```

Comment:
```suggestion
            `Unable to create integration policy. Integration '${pkgInfo.name}' already exists on this agent policy.`
```

the error refers to integration policy, not agent policy, so just a minor fix here